.evergreen/functions.yml

```diff
@@ -679,9 +679,8 @@ functions:
           eval $(.evergreen/print-compass-env.sh)
 
           if [[ "$IS_WINDOWS" == "true" ]]; then
-            # TODO: windows_setup
```

Comment:
I assume windows_setup referred to the .exe, i.e. COMPASS-8706?

Reply:
Oh 👀 You're right 👍